src/data/sql-quiz.ts

```diff
@@ -829,6 +829,28 @@ const sqlQuiz = [
     Explanation:
       "The BETWEEN operator is an intuitive way to use the WHERE clause to help narrow down a result set.",
     Link: "https://www.freecodecamp.org/news/a-beginners-guide-to-sql/"
+  },
+  {
+    Question:
+      "What SQL statement contains a valid subquery?",
+    Answer: "SELECT * FROM employees WHERE wage > (SELECT AVG(wage) FROM employees)",
+    Distractor1: "SELECT MAX(*) FROM employees",
+    Distractor2: "SELECT AVG(wage) FROM employees WHERE name IN ('name1', 'name2', 'name3')",
```

Comment:
this looks valid to me
we should alter this one to be one of the wrong answers

Reply:
Does that make it a good distractor? An in-clause is not a sub query, but I can still update this question if you would like me too!

Reply:
yeah, I think at first glance it looked valid but at second look that makes sense why it isn't.
we could leave it.

Some questions, will have more obviously wrong and right answers why others will take a few more seconds to look at to check for the right answer.

So it is good to have a mixture of both 👍